sqlmesh/cicd/bot.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import typing as t
+
+import click
+
+from sqlmesh.cli import error_handler
```

Comment:
one line, error_handler, options as opt

Reply:
isort forces it back. Also I think whatever import pattern preferences we have should be enforced by our tooling.